src/index.ts

```diff
@@ -1,10 +1,11 @@
-import { TinyEmitter } from 'tiny-emitter';
-import Metrics from './metrics';
-import type IStorageProvider from './storage-provider';
-import LocalStorageProvider from './storage-provider-local';
-import InMemoryStorageProvider from './storage-provider-inmemory';
+import { TinyEmitter } from "tiny-emitter";
+import Metrics from "./metrics";
+import { boostrapData } from "./boostrapData";
+import type IStorageProvider from "./storage-provider";
+import LocalStorageProvider from "./storage-provider-local";
+import InMemoryStorageProvider from "./storage-provider-inmemory";
```

Comment:
Please use single quote

Reply:
That's the prettier effect. I'll change it.